README.md

```diff
@@ -132,6 +132,7 @@ Before working with SDO data make sure to checkout the [Guide to SDO Data Analys
 - Angryk, Rafal A., et al. "Multivariate time series dataset for space weather data analytics." Scientific data 7.1 (2020): 1-13. [link](https://www.nature.com/articles/s41597-020-0548-x.pdf) [code](https://bitbucket.org/gsudmlab/workspace/projects/FP)
 - Shneider, Carl, et al. "A Machine-Learning-Ready Dataset Prepared from the Solar and Heliospheric Observatory Mission." arXiv preprint arXiv:2108.06394 (2021). [link](https://arxiv.org/abs/2108.06394)
 - Baek, Ji-Hye, et al. "Solar Event Detection Using Deep-Learning-Based Object Detection Methods." Solar Physics 296.11 (2021): 1-15. [link](https://link.springer.com/article/10.1007/s11207-021-01902-5)
+- McGranaghan, Ryan; ryan.mcgranaghan@colorado.edu; https://orcid.org/0000-0002-9605-0007; Mannucci, Anthony; http://orcid.org/0000-0003-2391-8490; Mattmann, Chris; et al. (2018): Machine learning databases used for Journal of Geophysical Research: Space Physics manuscript: "New capabilities for prediction of high-latitude ionospheric scintillation: A novel approach with machine learning.". figshare. Dataset. https://doi.org/10.6084/m9.figshare.6813131.v1 
```

Comment:
please convert links to markdown links and use the MLA style for the citation

Reply:
McGranaghan, R., et al. "Machine learning databases used for Journal of Geophysical Research: Space Physics manuscript: New capabilities for prediction of high-latitude ionospheric scintillation: A novel approach with machine learning.". (2018). figshare. Dataset. [link](https://doi.org/10.6084/m9.figshare.6813131.v1)